doc/ref_arch/openstack/refs.bib

```diff
@@ -1,837 +1,649 @@
+}
+
+@misc{openstackpol,
+	title = {{OpenStack Security Guide - Policies}},
+	url = {https://docs.openstack.org/security-guide/identity/policies.html#policy-section}
+}
+
+@misc{openstackdr,
+	title = {{OpenStack KeyStone Default Roles}},
+	url = {https://docs.openstack.org/keystone/latest/admin/service-api-protection.html}
+}
+
+@misc{openstackseccom,
+	title = {{OpenStack - Introduction to TLS and SSL}},
+	url = {https://docs.openstack.org/security-guide/secure-communication/introduction-to-ssl-and-tls.html}
+}
+
+@misc{ciscat,
+	title = {{Center for Internet Security CIS-CAT Pro}},
+	url = {https://www.cisecurity.org/cybersecurity-tools/cis-cat-pro/}
+}
+
+@misc{cisben,
+	title = {{Center for Internet Security Benchmarks}},
+	url = {https://www.cisecurity.org/cis-benchmarks/}
+}
+
+@misc{openstackisv,
+	title = {{OpenStack Image Signature Verification}},
+	url = {https://docs.openstack.org/glance/wallaby/user/signature.html}
+}
+
+@misc{openstacksr,
+	title = {{OpenStack - SR-IOV Passthrough For Networking}},
+	url = {https://wiki.openstack.org/wiki/SR-IOV-Passthrough-For-Networking}
+}
+
+@misc{openstackti,
+	title = {{OpenStack Trusted Images}},
+	url = {https://docs.openstack.org/security-guide/instance-management/security-services-for-instances.html#trusted-images}
+}
+
+@misc{openstackim,
+	title = {{OpenStack Virtual Machine Image Guide}},
+	url = {https://docs.openstack.org/image-guide/}
+}
+
+@misc{openstackasi,
+	title = {{Adding Signed Images}},
+	url = {https://docs.openstack.org/operations-guide/ops-user-facing-operations.html#adding-signed-images}
+}
+
+@misc{etsisol4,
+	title = {{Network Functions Virtualisation (NFV) Release 4; Protocols and Data Models; VNF Package and PNFD Archive specification}},
+	howpublished = {ETSI GS NFV-SOL 004 V4.3.1},
+	url = {https://www.etsi.org/deliver/etsi_gs/NFV-SOL/001_099/004/04.03.01_60/gs_NFV-SOL004v040301p.pdf},
+	year = 2022,
+	month =	 jul
+}
+
+@misc{etsisec21,
+	title = {{Network Functions Virtualisation (NFV) Release 2; Security; VNF Package Security Specification}},
+	howpublished = {ETSI GS NFV-SEC 021 V2.6.1},
+	url = {https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/021/02.06.01_60/gs_nfv-sec021v020601p.pdf},
+	year = 2019,
+	month =	 jun
+}
+
+% Chapter 7 References
+
+@misc{forem,
+	title = {{Foreman}},
+	url = {https://www.theforeman.org/}
+}
+
+@misc{ansib,
+	title = {{Ansible Documentation}},
+	url = {https://docs.ansible.com/}
+}
+
+@misc{tripl,
+	title = {{OpenStack TripleO}},
+	url = {http://opendev.org/openstack/tripleo-common}
+}
+
+@misc{trarch,
+	title = {{OpenStack TripleO Architecture}},
+	url = {https://docs.openstack.org/tripleo-docs/latest/install/introduction/architecture.html#project-architecture}
+}
+
+@misc{airsh,
+	title = {{Airship v2}},
+	url = {https://www.airshipit.org/}
+}
+
+@misc{starl,
+	title = {{StarlingX - Deploy Your Edge Cloud Now}},
+	url = {https://www.starlingx.io/}
+}
+
+% Chapter 8 References
```

Comment:
The RC Chapter 8 has quite a few references -- should we include and label this Chapter 9